students/luke/s01/schedule.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+"""
+Schedule students for lightning talks, winter 2018
+"""
+import random
+
+students = open('students.txt').readlines()
+
+# remove the header line
+del students[0]
+
+# strip the whitespace
+students = [line.strip() for line in students]
+
+# remove the languages, colon, etc.
+students = [line.split(":")[0] for line in students]
+
+# reverse the first, last names
+# separate them:
+#students = [line.split(",") for line in students]
+# put them back together
+#students = ["{} {}".format(first.strip(), last) for last, first in students]
+
+# put them in random order
+random.shuffle(students)
+
+# make a list from 1 to 10
+weeks = list(range(2, 11))
```

Comment:
You could also turn this into a function: 

def num_list(start, stop):
    out_list = list(range(start, stop +1))
    return(out_list)

That way you are always including the stop number, without having to manually code it.